h/static/styles/annotations.scss

```diff
@@ -162,3 +212,25 @@ privacy {
     color: $text-color;
   }
 }
+
+.more, .less {
+  color: rgba(189, 28, 43, 0.5);
+  background: #fff;
+  font-style: italic;
+  font-family: Georgia,"Bitstream Charter","Times","Times New Roman",serif;
```

Comment:
This should probably be from one of our font variables rather than an explicit set of families unique to this component.